README.md

```diff
@@ -2,15 +2,15 @@ This branch is work in progress for bringing grape-swagger to Swagger 2.0 spec.
 
 ##### Table of Contents
 
-[What is grape-swagger?](#what)  
-[Related Projects](#related)  
-[Swagger-Spec](#swagger-spec)  
-[Installation](#install)  
-[Usage](#usage)  
-[Configure](#configure)  
-[Routes Configuration](#routes)  
-[Additional documentation](#additions)  
-[Example](#example)  
+[What is grape-swagger?](#what)
+[Related Projects](#related)
+[Swagger-Spec](#swagger-spec)
+[Installation](#install)
+[Usage](#usage)
+[Configure](#configure)
+[Routes Configuration](#routes)
+[Additional documentation](#additions)
+[Example](#example)
```

Comment:
My editor removes trailing lines too, ... can you turn this into a list or undo this one? The file is no longer readable.